build/base/intel-builder.Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN apt update \
     && apt install -y --no-install-recommends \
         libstdc++-12-dev binutils procps clang \
         intel-oneapi-compiler-dpcpp-cpp \
-        intel-oneapi-mpi-devel \
+        intel-oneapi-mpi-devel-2021.13 \
```

Comment:
I'm ok with this as a temporary solution, but eventually this version will be out of support or have CVEs, so we still need to figure what's wrong with newer versions.

WDYT @tenzen-y?

Reply:
I did not find the exhaustive changelog for release 2021.14, but it seems they mention that they changed the default IPC exchange mechanism to be pidfd instead of drmfd which was the default. It's not clear to me if that could be the issue.

Reply:
Thank you for this great investigation!
I'm ok with this solution as a short term. Could we open an issue to upgrade to 2021-.14?

Reply:
I'm curious about the reason why we face this issue only in the github actions, though.

Reply:
https://github.com/kubeflow/mpi-operator/issues/678